src/blocks/Sumcheck.sol

```diff
@@ -40,7 +40,7 @@ library PrimarySumcheck {
             uint256 r_i;
             label[0] = 99; // c_label[0] = 99;
             (transcript, r_i) = KeccakTranscriptLib.squeeze(transcript, ScalarFromUniformLib.curveVesta(), label);
-
+            r_i = Field.reverse256(r_i);
```

Comment:
It looks like in the definition of `squeeze` you call `Field.reverse256` before outputting `r_i` Unless I'm misunderstanding, could you delete both instances of `Field.reverse256` and arrive at the same result?

Reply:
nice catch! Thanks